.github/workflows/cd-deploy-nodes-gcp.yml

```
@@ -225,14 +143,14 @@ jobs:
   #   otherwise a new major version is deployed in a new MiG.
   #
   # Runs:
-  # - on every push/merge to the `main` branch
+  # - on every push to the `main` branch
   # - on every release, when it's published
   deploy-nodes:
     strategy:
       matrix:
         network: [Mainnet, Testnet]
     name: Deploy ${{ matrix.network }} nodes
-    needs: [ build, test-configuration-file, versioning ]
+    needs: [ build, versioning, test-configuration-file, test-configuration-file-testnet, test-zebra-conf-path ]
```

We can't depend on testnet jobs because testnet is unreliable.

```suggestion
    needs: [ build, versioning, test-configuration-file, test-zebra-conf-path ]
```